renderdoc/core/target_control.cpp

```diff
@@ -242,6 +249,16 @@ void RenderDoc::TargetControlClientThread(uint32_t version, Network::Socket *cli
         }
       }
     }
+    else if(prevWindows != curWindows)
+    {
+      prevWindows = curWindows;
+
+      WRITE_DATA_SCOPE();
```

Comment:
This needs to be protected by a version check, otherwise if an older version of the renderdoc controller connected to a newer target, it would send this packet and the controller would disconnect as it doesn't recognise it.